lib/engines/content_object_store/app/components/content_object_store/content_block/document/show/summary_list_component.rb

```diff
@@ -14,6 +14,8 @@ def items
       organisation_item,
       creator_item,
       embed_code_item,
+      state_item,
+      scheduled_item,
```

Comment:
I think it'd be good to only show this row if a block is still scheduled

Reply:
Tom did bring this up as well, but my argument was why spend time writing logic and tests for a UI that we haven't finalised yet? I wanted it to be as crap as possible to ensure we return to it 😆 but aware that probably isn't great practice so happy to change!

Reply:
Yeah we could go either way with this but given we are skipping ahead with the build I like what we did with embed code: we just popped it on the page without overthinking it and our message to the team was this is here for us, help us make it better.

Reply:
Yeah, that's fair